src/NzbDrone.Core/Notifications/Telegram/Telegram.cs

```diff
@@ -69,5 +74,37 @@ public override ValidationResult Test()
 
             return new ValidationResult(failures);
         }
+
+        private string FormatMessageWithLink(string message, Series series)
```

Comment:
This should be renamed to `GetLink` and just return the link, which gets passed to `SendNotification` and the message is formatted there.